pkg/features/list.go

```diff
@@ -134,6 +134,9 @@ var (
 
 	// Adds built-in policy to ensure that Red Hat images are signed by Red Hat Release Key
 	RedHatImagesSignedPolicy = registerFeature("Adds built-in policy to ensure that Red Hat images are signed by the Red Hat release key", "ROX_RED_HAT_IMAGES_SIGNED_POLICY", unchangeableInProd)
+
+	// Locks process baselines when their deployments leave the observation period
+	AutolockAllProcessBaselines = registerFeature("Locks process baselines when their deployments leave the observation period", "ROX_AUTOLOCK_ALL_PROCESS_BASELINES")
```

Comment:
Nit:  I know it is ALL now, but if we do the config changes to limit it later we may still use the flag.  So I would remove the work all.  But that is just my opinion.

```suggestion
	AutoLockProcessBaselines = registerFeature("Locks process baselines when their deployments leave the observation period", "ROX_AUTOLOCK_PROCESS_BASELINES")
```